supabase/seed.sql

```diff
@@ -0,0 +1,7 @@
+insert into public.access_points_latest_states
```

Comment:
This is nice! Sadly we can only have a single Kafka connection running at a time, which is used to update the `access_points_latest_states` in the production database. Should we add a local cron job that syncs this table from production into the local database instance every 5 minutes? So you have representative data and behaviour locally?

Reply:
That sounds a bit like an overkill for me. I'm not even sure that we have to actually make the local app work with the local database in normal circumstances (or only use the local database to automate this migration & type thing). If we need real-life data, we could very well connect to a production database probably since we only need to read stuff? The other thing would however be to have representative data in both prod & `develop` databases :( If we have to solve that problem there then yeah, maybe we can apply the same solution locally.